dropwizard-jobs-core/src/main/java/de/spinscale/dropwizard/jobs/DropwizardJobFactory.java

```diff
@@ -1,11 +1,13 @@
 package de.spinscale.dropwizard.jobs;
 
-import org.quartz.Job;
```

Comment:
This class must now accept  `de.spinscale.dropwizard.jobs.Job` instances as the constructor argument. It's not a breaking change, because the class that instantiates the factory already uses them...